databuilder/transformer/bigquery_usage_transformer.py

```diff
@@ -0,0 +1,38 @@
+from databuilder.transformer.base_transformer import Transformer
+from databuilder.models.table_column_usage import ColumnReader, TableColumnUsage
+from databuilder.extractor.bigquery_usage_extractor import TableColumnUsageTuple
+
+
+class BigqueryUsageTransformer(Transformer):
+    """
+    Transformer to convert TableColumnUsageTuple data to bigquery usage data
+    which can be uploaded to Neo4j
+    """
+    def init(self, conf):
+        # type: (ConfigTree) -> None
+        self.conf = conf
+
+    def transform(self, record):
+        # type: (dict) -> Optional[TableColumnUsage]
+        if not record:
+            return None
+
+        (key, count) = record
+
+        if not isinstance(key, TableColumnUsageTuple):
+            raise Exception("BigqueryUsageTransformer expects record of type TableColumnUsageTuple")
+
+        col_readers = []
+        col_readers.append(ColumnReader(database=key.database,
+                                        cluster=key.cluster,
+                                        schema=key.schema,
+                                        table=key.table,
+                                        column=key.column,
```

Comment:
so column will be * ?

Reply:
Yes, I followed the same convention as the sql_to_table col usage transformer.